modules/contact-form/blocks/form/block.jsx

```diff
@@ -0,0 +1,95 @@
+const {
+	registerBlockType,
+	InspectorControls,
+	InnerBlocks,
+} = wp.blocks;
+
+const { createElement } = wp.element;
+const { CheckboxControl, TextControl } = wp.components;
+
+const { __ } = wp.i18n;
+
+// Define the form block
+let Form = {
+	title: __( 'Form' ),
+	icon: 'feedback',
+	category: 'common',
+	description: __( 'Contact Form Settings' ),
+	keywords: [
+		__( 'form' ),
+		__( 'contact' ),
+	],
+	useOnce: true,
+	attributes: {
+		subject: {
+			type: 'string',
+			default: 'Feedback',
+		},
+
+		to: {
+			type: 'string',
+			default: ''
+		}
+	},
+
+	save: ( { attributes, className } ) => {
+		return <div className={ className }>
+			<form>
+				<InnerBlocks.Content />
+			</form>
+		</div>
+	},
+
+	edit: ( { attributes, setAttributes, className, isSelected } ) => {
+		return [
+			// Display the inner blocks
+			<InnerBlocks
+				template={[
+					[ 'jetpack/form-text', {
+						'label': __( 'Name' ),
+						'placeholder': __( 'Full Name' ),
+						'required': true,
+					} ],
+					[ 'jetpack/form-text', {
+						'label': __( 'Email' ),
+						'placeholder': __( 'example@example.com' ),
+						'required': true,
+					} ],
+					[ 'jetpack/form-textarea', {
+						'label': __( 'Message' ),
+						'placeholder': __( 'Enter message' ),
+						'required': true,
+					} ],
+					[ 'jetpack/form-button', { 'label': __( 'Send' ) } ]
+				]}
+				allowedBlocks={ [
+					'jetpack/form-text',
+					'jetpack/form-textarea',
+					'jetpack/form-button'
+				] }
+			/>,
+
+			// Display on Focus
+			!! isSelected &&
```

Comment:
We can remove this check. https://github.com/WordPress/gutenberg/pull/5029 made `<InspectorControls>` check for `isSelected` automatically.